### DISCUSSION TITLE
weekly/2024-08-18

### DISCUSSION THREAD

**Discussion by giscus** [ATQQ/sugar-blog#286] [Announcements]
# weekly/2024-08-18

① 安全赋值运算符提案 ② 英：45个提升生产力的VS Code快捷指令 ③ 文字徽标制作工具 ④ Rslib -  ⑤ 使用js实现打字机效果 ⑥ Aide - 在 VSCode 中征服任何代码 ⑦ Product Hunt 每日中文热榜

https://sugarat.top/weekly/2024-08-18.html

**Comment by XiaofengdiZhu** [comment]
Rslib的系列产品里有个Rspress，感觉做得不错

**Reply by ATQQ** [reply]
√，Rspack支持 VItePress 画风 React版
